src/bls.ts

```diff
@@ -1235,32 +1409,41 @@ async function doSignProcessOneImpl( i, optsSignOperation ) {
         log.generateTimestampString( null, true ), "skale_imaVerifyAndSign", i, strNodeURL,
         optsSignOperation.fromChainName, optsSignOperation.targetChainName,
         joParams, optsSignOperation.sequenceId );
-    const joIn = { "method": "skale_imaVerifyAndSign", "params": joParams };
+    const joIn: any = { method: "skale_imaVerifyAndSign", params: joParams };
     const joOut = await joCall.call( joIn );
     await doSignProcessHandleCall( optsSignOperation, joParams, joCall, joIn, joOut, i );
 }
 
 async function doSignMessagesImpl(
-    nTransferLoopCounter, strDirection,
-    jarrMessages, nIdxCurrentMsgBlockStart, strFromChainName,
-    joExtraSignOpts, fn
+    nTransferLoopCounter: number, strDirection: string,
+    jarrMessages: any[], nIdxCurrentMsgBlockStart: number, strFromChainName: string,
+    joExtraSignOpts?: loop.TExtraSignOpts | null, fn?: IMA.TFunctionAfterSigningMessages
 ) {
-    const optsSignOperation = {
+    const optsSignOperation: TSignOperationOptions = {
         imaState: state.get(),
-        nTransferLoopCounter: nTransferLoopCounter,
-        strDirection: strDirection,
-        jarrMessages: jarrMessages,
-        nIdxCurrentMsgBlockStart: nIdxCurrentMsgBlockStart,
-        strFromChainName: strFromChainName,
-        joExtraSignOpts: joExtraSignOpts,
-        fn: fn,
+        nTransferLoopCounter,
+        strDirection,
+        jarrMessages,
+        nIdxCurrentMsgBlockStart,
+        strFromChainName,
+        joExtraSignOpts,
+        // eslint-disable-next-line n/handle-callback-err
```

Comment:
Why do we need to disable linter here?

Reply:
Yes, because no way to avoid it in async function declaration